instrumentation/couchbase/couchbase-2.6/javaagent/src/test/groovy/CouchbaseSpanUtil.groovy

```diff
@@ -37,16 +37,16 @@ class CouchbaseSpanUtil {
         "$SemanticAttributes.NET_TRANSPORT" { it == null || it == IP_TCP }
         // Because of caching, not all requests hit the server so these attributes may be absent
         "net.sock.peer.addr" { it == "127.0.0.1" || it == null }
-        "net.sock.peer.name" { it == "localhost" || it == null }
-        "net.sock.peer.port" { it == null || Number }
+        "net.sock.peer.name" { it == "localhost" || it == "127.0.0.1" || it == null }
```

Comment:
An alternative would be to add `&& (sockPeerAddr == null || !sockPeerAddr.equals(sockPeerName))` to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/7d01a221a6678b35b7409af3ba7b745a6b3fa461/instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/net/NetClientAttributesExtractor.java#L76

Reply:
Agree. @lmolkova do you think it makes sense to include this additional condition in the spec?

Reply:
@mateuszrzeszutek sure!
Just to clarify: `net.sock.peer.addr` should only be set if not the same as `net.sock.peer.name`, correct?

It's implicitly in the spec, but I agree it can be clarified and mentioned in yaml file too:

> If peer information available to instrumentation can represent DNS name or IP address, instrumentation SHOULD NOT attempt to parse it and SHOULD only set net.sock.peer.name.

I can add a new condition to `net.sock.peer.name` requirement level like 
recommended: If available and different from `net.peer.name` **and `net.sock.peer.addr`** and if `net.sock.peer.addr` is set

Reply:
> recommended: If available and different from `net.peer.name` **and `net.sock.peer.addr`** and if `net.sock.peer.addr` is set

Sounds perfect, thanks!